medcat/cat.py

```diff
@@ -1105,7 +1105,7 @@ def get_entities_multi_texts(self,
                             out.insert(i, self._doc_to_out(None, only_cui, addl_info))  # type: ignore
 
                 cnf_annotation_output = self.config.annotation_output
-                if not(cnf_annotation_output.include_text_in_output):
+                if not (cnf_annotation_output.include_text_in_output):
```

Comment:
why does this expression need brackets?

Reply:
You're right - better to remove that altogether :)
But I just fixed what flake8 complained about.